modules/stochastic_tools/src/distributions/LognormalDistribution.C

```diff
@@ -0,0 +1,32 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+#include "LognormalDistribution.h"
+
+template <>
+InputParameters
+validParams<LognormalDistribution>()
+{
+  InputParameters params = validParams<Distribution>();
+  params.addClassDescription("Boost Lognormal distribution.");
+  params.addRequiredParam<Real>("location", "The Lognormal location parameter.");
+  params.addRequiredParam<Real>("scale", "The Lognormal scale parameter.");
```

Comment:
changes these to be 'standard_deviation' and 'mean', make sure it matches what is passed to boost.

Reply:
Boost still calls these location and scale for lognormal distribution. Both (mean, standard deviation) and (location, scale) are valid for normal distribution.